.github/workflows/gradle-build-deploy.yml

```diff
@@ -29,3 +29,20 @@ jobs:
       with:
         name: website
         path: build/site
+        
+
+  deploy:
+    needs:
+      - release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS credentials from Production account
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
These creds, how much does the user/role have access to? May want to make a new role with only access to the specific prefix in the s3 bucket to isolate breaches if they happen.